crates/sequencing/papyrus_consensus/src/config.rs

```diff
@@ -45,6 +48,12 @@ pub struct ConsensusConfig {
 impl SerializeConfig for ConsensusConfig {
     fn dump(&self) -> BTreeMap<ParamPath, SerializedParam> {
         let mut config = BTreeMap::from_iter([
+            ser_param(
+                "chain_id",
+                &self.chain_id,
+                "The chain id of the Starknet chain.",
```

Comment:
Could be nice to have a couple of examples like mainnet/sepolia. (I don't know if this is something we usually do in config descriptions).

Reply:
I added documentation for details.
<!-- Reviewable comment -ODKvyYlCKn9FYTJmX4f:bw18ld1 -->
<!-- Sent from Reviewable.io -->